Druid-Tests/DRDispatchTableBuilderTest.class.st

```diff
@@ -28,7 +28,7 @@ DRDispatchTableBuilderTest >> jitCompilerClassForTest [
 DRDispatchTableBuilderTest >> setUp [
 
 	super setUp.
-	interpreter :=	DRInterpreterToCompiler fromInterpreter: (self interpreterWith: #(#primitiveAdd #primitiveEqual)).
+	interpreter :=	DRInterpreterToCompiler fromInterpreterBuilder: (self interpreterWith: #(#primitiveAdd #primitiveEqual)).
```

Comment:
I do not particularly like the "Interpreter builder" API.
We can keep it for now, but we should change it in the future.